catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/DurableFileSystemFileConsumer.java

```diff
@@ -108,5 +108,9 @@ private AsyncFileAlterationObserver backwardsCompatibility(String fileName) {
   public void shutdown() throws Exception {
     super.shutdown();
     listener.destroy();
+
+    if (observer != null) {
+      observer.destroy();
```

Comment:
❗ Because you re-used the same `destroy()` method that already existed on the observer, the side effect is that the file will also be deleted. is that what you want?

Reply:
Hmm, this should be fine. This destroy method will delete the root folder, which should be safe to get rid of if the object is being destroyed. Everything that accesses that variable is ultimately called by DurableFileSystemFileConsumer, so sincee we're shutting it down in that class I can't see it needing to be used again